census/src/main/scala/grasshopper/census/search/AddressInterpolator.scala

```diff
@@ -91,12 +59,48 @@ object AddressInterpolator {
     Feature(schema, values)
   }
 
+  private def prefix(f: Feature, a: Int): String = {
+    val addressIsEven = a % 2 == 0
+    val rightRangeIsEven: Boolean = rangeIsEven(f, false)
+
+    val leftRangeIsEven: Boolean = rangeIsEven(f, true)
+
+    if (addressIsEven && rightRangeIsEven)
+      "R"
+    else if (addressIsEven && leftRangeIsEven)
+      "L"
+    else if (!addressIsEven && !rightRangeIsEven)
+      "R"
+    else if (!addressIsEven && !leftRangeIsEven)
+      "L"
+    else
+      "R"
```

Comment:
Is there a legit scenario where it _should_ fall through and default to "R"?  If yes, seem comment-worthy.  If no, seems like this should either raise an Exception or log a `WARN`-level message before returning "R".  Either option should have the full details of the `Feature` so we can track down _why_ we can't determine which side of the street a given address is on.